src/libs/DistanceRequestUtils.ts

```diff
@@ -292,17 +292,17 @@ function convertToDistanceInMeters(distance: number, unit: Unit): number {
 function getCustomUnitRateID(reportID: string) {
     const report = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
     const parentReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${report?.parentReportID}`];
-    const policy = PolicyUtils.getPolicy(report?.policyID ?? parentReport?.policyID ?? '-1');
-    let customUnitRateID: string = CONST.CUSTOM_UNITS.FAKE_P2P_ID;
+    const policy = getPolicy(report?.policyID ?? parentReport?.policyID);
+    let customUnitRateID: string | undefined = CONST.CUSTOM_UNITS.FAKE_P2P_ID;
```

Comment:
Hm, maybe we should default the value to `CONST.CUSTOM_UNITS.FAKE_P2P_ID` instead of letting it be undefined?